includes/Register_Font.php

```diff
@@ -0,0 +1,69 @@
+<?php
+/**
+ * Class Register_Font.
+ *
+ * @package   Google\Web_Stories
+ * @copyright 2020 Google LLC
+ * @license   https://www.apache.org/licenses/LICENSE-2.0 Apache License 2.0
+ * @link      https://github.com/google/web-stories-wp
+ */
+
+/**
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+namespace Google\Web_Stories;
+
+/**
+ * Class Register_Font
+ *
+ * @package Google\Web_Stories
+ */
+class Register_Font {
+	/**
+	 * Font Style handle.
+	 *
+	 * @var string
+	 */
+	const STYLE_HANDLE = 'google-fonts';
```

Comment:
Should this be `web-stories-fonts` instead, since the plugin's slug is recommended generally as the prefix?

Reply:
The handle is prefixed using `get_handle`. See comment below.